datajunction-server/datajunction_server/database/attributetype.py

```diff
@@ -87,7 +87,10 @@ class ColumnAttribute(Base):
     """
 
     __tablename__ = "columnattribute"
-    __table_args__ = (UniqueConstraint("attribute_type_id", "column_id"),)
+    __table_args__ = (
+        UniqueConstraint("attribute_type_id", "column_id"),
+        Index("idx_columnattribute_column_id", "column_id"),
```

Comment:
I think that if the earlier UniqueConstraint had the column order switched then we wouldn't need this new one at all.
If it's easier to add then to change the other one then go for it.

Reply:
I ended up removing this because I'm not sure it makes much of a difference